bin/download_extract_rc.ps1

```diff
@@ -0,0 +1,89 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+# Helper script for lazy Windows Developers
+# Downloading, checking and extracting the sources of a rc candidate for Windows
+
+<#
+.SYNOPSIS
+    Little helper script for lazy Windows (Release) Developers
+.DESCRIPTION
+    This command downloads, checks and extract a CouchDB source tarball
+
+    -CouchDBVersion         CouchDB version number to download (default 3.4.2)

```

Comment:
Does it make sense to make it default to any value?  Unless you are planning to keep that default value updated on each new release, it can get more and more misleading as the time goes by.

Reply:
that sounds logical, updated.